tests/components/glances/__init__.py

```diff
@@ -12,168 +12,6 @@
     "verify_ssl": True,
 }
 
-MOCK_DATA = {
```

Comment:
Why do we remove this?

Reply:
It isn't used anymore since the unit tests moved to snapshots a few months ago.